meldekortdomene/src/main/kotlin/no/nav/aap/varsel/SendVarselJobbUtfører.kt

```diff
@@ -0,0 +1,49 @@
+package no.nav.aap.varsel
+
+import no.nav.aap.komponenter.dbconnect.DBConnection
+import no.nav.aap.komponenter.miljo.Miljø
+import no.nav.aap.komponenter.repository.RepositoryRegistry
+import no.nav.aap.lookup.gateway.GatewayProvider
+import no.nav.aap.motor.Jobb
+import no.nav.aap.motor.JobbInput
+import no.nav.aap.motor.JobbUtfører
+import no.nav.aap.motor.cron.CronExpression
+import java.time.Clock
+
+class SendVarselJobbUtfører(
+    private val varselService: VarselService,
+) : JobbUtfører {
+    override fun utfør(input: JobbInput) {
+        if (!Miljø.erProd()) {
```

Comment:
Hvorfor ikke unleash?

Reply:
Unleash er ikke lagt inn her fra før, så for å spare tid bare 😅